### PR TITLE
Extracting strings to i18n file

### LINKED ISSUE #19: Internationalisation
This issue is related to internationalisation of Carator app. To close this issue, app should be internationalised using [i18n](https://guides.rubyonrails.org/i18n.html) gem.

### PR DESCRIPTION
**Pull request by wzieba**
Updating pg to 1.1.3

Closes #19

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.5.1
```
Comment (wzieba) [inline .ruby-version]: This file has been generated by `rbenv` and I think it should be included in repo

File: db/schema.rb [hunk 2]
```diff
@@ -50,4 +47,5 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
   end
 
+  add_foreign_key "cars", "users"
```
Comment (wzieba) [inline db/schema.rb]: Not sure why it has been added nor why application worked previously without this line

File: db/schema.rb [hunk 3]
```diff
@@ -18,18 +18,15 @@
...
-    t.integer "user_id"
+    t.bigint "user_id"
+    t.integer "production_year"
     t.string "trim"
     t.string "custom_name"
     t.index ["user_id"], name: "index_cars_on_user_id"
   end
 
-  # These are extensions that must be enabled in order to support this database
-  enable_extension "plpgsql"
```
Comment (wzieba) [inline db/schema.rb]: duplicated https://github.com/danille/carator/pull/20/files#diff-1acd2e7e27a227829d5d14a91c863bb6L15

File: features/step_definitions/cars_garage_steps.rb [hunk 4]
```diff
@@ -27,6 +27,7 @@
 
 When("I submit filled in new car form") do
   fill_in 'car_custom_name', with: "Test car custom name"
+  sleep 2
```
Comment (wzieba) [inline features/step_definitions/cars_garage_steps.rb]: Mind ⚠️

Comment (danille) [reply features/step_definitions/cars_garage_steps.rb]: "How to speed up your app? Just remove `sleep` you've put in code before" :)

File: db/schema.rb [hunk 5]
```diff
@@ -18,18 +18,15 @@
   create_table "cars", force: :cascade do |t|
     t.string "manufacturer"
     t.string "model"
-    t.integer "production_year"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.integer "user_id"
+    t.bigint "user_id"
```
Comment (danille) [inline db/schema.rb]: Is there any real need to change `user_id` type to `bigint`?

File: features/step_definitions/cars_garage_steps.rb (hunk 4 above)
Comment (danille) [inline features/step_definitions/cars_garage_steps.rb]: What purpose has this `sleep` been added for?

File: db/schema.rb (hunk 2 above)
Comment (danille) [inline db/schema.rb]: Was it generated after `pg` gem update?

File: Gemfile.lock [hunk 6]
```diff
@@ -319,7 +319,7 @@ DEPENDENCIES
   jbuilder (~> 2.5)
   jquery-rails (= 4.3.1)
   listen (>= 3.0.5, < 3.2)
-  pg
+  pg (~> 1.1.3)
```
Comment (danille) [inline Gemfile.lock]: Was you intention to specify particular version with patch number or just major.minor? If first, you should specify version directly, without `~>` operator. More info [here](https://stackoverflow.com/questions/5101591/what-does-mean-in-a-gem-file)

**Review by danille — CHANGES_REQUESTED** [review]

File: db/schema.rb (hunk 2 above)
Comment (wzieba) [reply db/schema.rb]: yes, generated on `bundle install`

File: db/schema.rb (hunk 5 above)
Comment (wzieba) [reply db/schema.rb]: generated on `bundle install`, probably `pg 1.1.3` update had something in common

File: features/step_definitions/cars_garage_steps.rb (hunk 4 above)
Comment (wzieba) [reply features/step_definitions/cars_garage_steps.rb]: to wait for load async job of js script for choosing year. Passing of this test previously was matter of luck, you can check that you have to always wait a little to load years. Sometimes the time is short, sometimes long enough to break test

**Comment by wzieba** [comment]
Improvements committed, questions answered @danille

**Review by danille — APPROVED** [review]

**Pull request merged**